retrieval/target_test.go

```diff
@@ -293,7 +293,7 @@ func TestTargetScrapeTimeout(t *testing.T) {
 	)
 	defer server.Close()
 
-	testTarget := newTestTarget(server.URL, 25*time.Millisecond, clientmodel.LabelSet{})
```

Comment:
Shouldn't this stay at 25, but we just sleep longer to have a higher chance of hitting the deadline?